test/e2e/suite_test.go

```diff
@@ -94,10 +91,8 @@ func init() {
 
 func TestE2E(t *testing.T) {
 	RegisterFailHandler(Fail)
-	junitPath := path.Join(artifactFolder, fmt.Sprintf("junit.e2e_suite.%d.xml", config.GinkgoConfig.ParallelNode))
-	junitReporter := reporters.NewJUnitReporter(junitPath)
 
-	RunSpecsWithDefaultAndCustomReporters(t, "capi-ibmcloud-e2e", []Reporter{junitReporter})
+	RunSpecs(t, "capi-ibmcloud-e2e")
```

Comment:
can elaborate more on this change and also possible the makefile change and add the content to the PR description for removing the junitreporter from the code.